### PR TITLE
Add trailing commas to some parameter lists to get better formatting

### PR DESCRIPTION
**Pull request by chenzhiguang**

### CODE REVIEW

**Comment by chenzhiguang** [comment]
Hi @srawlins, please close https://github.com/dart-lang/markdown/pull/419 and review this PR instead. Thanks!

File: tool/stats_lib.dart [hunk 1]
```diff
@@ -15,12 +15,15 @@ import '../test/util.dart';
...
 // Locate the "tool" directory. Use mirrors so that this works with the test
 // package, which loads this suite into an isolate.
-String get toolDir =>
-    p.dirname((reflect(loadCommonMarkSections) as ClosureMirror)
-        .function
-        .location!
-        .sourceUri
-        .path);
+String get toolDir {
+  final path = (reflect(loadCommonMarkSections) as ClosureMirror)
```
Comment (srawlins) [inline tool/stats_lib.dart:19]: +1 thanks!

File: tool/update_blns.dart [hunk 2]
```diff
@@ -12,11 +12,9 @@ Future<void> main() async {
   try {
     final request = await client.getUrl(Uri.parse(_blnsJsonRawUrl));
     final response = await request.close();
-    json = (jsonDecode(await response
-            .cast<List<int>>()
-            .transform(utf8.decoder)
-            .join('')) as List)
-        .cast<String>();
+    final source =
```
Comment (srawlins) [inline tool/update_blns.dart:15]: +1 love this

**Review by srawlins — APPROVED** [review]
Thanks much!

**Pull request merged**